buku

```diff
@@ -200,208 +200,173 @@ class BukuCrypt:
             return hasher.digest()
 
     @staticmethod
-    def encrypt_file(iterations, dbfile=None):
+    def encrypt_file(iterations=8, dbfile=None, encfile=None, password=None, replace=True):
         """Encrypt the bookmarks database file.
 
         Parameters
         ----------
         iterations : int
-            Number of iterations for key generation.
+            Number of iterations for key generation. (Defaults to 8)
+        dbfile : str, optional
+            Custom database file path (including filename). Fallback value is the default DB.
+        encfile : str, optional
+            Encoded dbfile. (Defaults to dbfile + '.enc')
+        password : str, optional
+            Password to use (if not provided, will be prompted from the user).
+        replace : bool
+            If True (default), the original file will be removed on success.
+        """
+        BukuCrypt(iterations, dbfile, encfile, password, replace)._encrypt_file()
```

Comment:
Extracted initialization process into a constructor (it's identical for both encryption and decryption)